Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -20,6 +20,7 @@ public struct WebView: UIViewRepresentable {
         
         @Published var url: String
         let baseURL: String
+        let ajaxProvider = AjaxProvider()
```

Comment:
I'm not sure whether we need to specify access level`fileprivate` here or not because we are not using it for other variables, what do you think?